_posts/patterns/dropdown-menu-ex.html

```diff
@@ -0,0 +1,42 @@
+/**
+ * Starts the dropdown menu view example component.
+ */
+define([
+    'backbone',
+    'edx-ui-toolkit/js/dropdown-menu/dropdown-menu-view.js'
+    ],
+    function( Backbone, DropdownMenuView ) {
+        'use strict';
+
+        var dropdownModel = new Backbone.Model();
+
+        dropdownModel.set({
+            main: {
+                text: 'username',
+                url: 'dashboard'
+            },
+            button_label: 'User options dropdown',
+            items: [
+                {
+                    text: 'Dashboard',
+                    url: 'dashboard'
```

Comment:
Nit: it is a bit strange to have broken links in the example. I wonder if there's something that would make sense. Even just redirecting to '/' might be okay.